tools/boards.txt.py

```diff
@@ -621,6 +621,21 @@
         'serial': '921',
         'desc': [ 'Product page: https://www.wemos.cc/' ],
     }),
+    ( 'd1_wroom_02', {
+        'name': 'LOLIN(WEMOS) D1 ESP-WROOM-02',
+        'opts': {
+            '.build.board': 'ESP8266_WEMOS_D1WROOM02',
+            '.build.variant': 'd1_mini',
+            },
+        'macro': [
+            'resetmethod_nodemcu',
+            'flashmode_dio',
+            'flashfreq_26',
```

Comment:
`flashfreq_26` doesn't exist in macro list, only in a sub element. This causes boardsgen to fail. 
https://github.com/esp8266/Arduino/blob/c681587895c08944bcb4e84ba9ba86a24461f4a6/tools/boards.txt.py#L1092-L1113

Reply:
Adding this to the macros list fixes it:
```
'flashfreq_26': collections.OrderedDict([
        ( '.build.flash_freq', '26' ),
]),
```

Reply:
thanks @AronHetLam, I pushed a fix.